### PR TITLE
Duration

### PR DESCRIPTION
**Pull request by wayofthepie**
No altitude calculation yet, needs a bit more thought.

### CODE REVIEW

**Review by shanetreacy — APPROVED** [review]

File: src/main/java/ie/cit/teambravo/cardsec/CardsecApplication.java [hunk 1]
```diff
@@ -2,13 +2,16 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cache.annotation.EnableCaching;
 
+@EnableCaching
 @SpringBootApplication(scanBasePackageClasses = {
-        CardsecApplication.class
+		CardsecApplication.class
```
Comment (viciouspetal) [inline src/main/java/ie/cit/teambravo/cardsec/CardsecApplication.java:9]: not sure what's with your formatter but I don't like it. Too much whitespace.

File: src/main/java/ie/cit/teambravo/cardsec/services/ValidationServiceImpl.java [hunk 2]
```diff
@@ -27,7 +34,8 @@ public boolean validate(String panelId, String cardId, Boolean allowed) {
 		eventDto.setAccessAllowed(allowed);
 		eventDto.setTimestamp(System.currentTimeMillis());
 		eventDto.setLocation(panelLocatorService.getPanelLocation(panelId));
-
+		durationService.getTravelTimeBetween2Points(new LatLngAlt(36.12, -86.67, 10.0),
```
Comment (viciouspetal) [inline src/main/java/ie/cit/teambravo/cardsec/services/ValidationServiceImpl.java:37]: sth that I was thinking about: would it be worth to label these magic numbers? aka assign to a nice variable with a nice name?

**Review by viciouspetal — APPROVED** [review]

File: src/main/java/ie/cit/teambravo/cardsec/services/ValidationServiceImpl.java (hunk 2 above)
Comment (wayofthepie) [reply src/main/java/ie/cit/teambravo/cardsec/services/ValidationServiceImpl.java:37]: Oh they are just there so something happens for now when you call the API. Can drop them once we get data down from the controller.

File: src/main/java/ie/cit/teambravo/cardsec/CardsecApplication.java (hunk 1 above)
Comment (wayofthepie) [reply src/main/java/ie/cit/teambravo/cardsec/CardsecApplication.java:9]: Ya can't figure out why... Using the ecliose formatter. It looks fine in intellij

**Pull request merged**